### DISCUSSION TITLE
kubectl-stern doesn't work for me

### DISCUSSION THREAD

**Discussion by asheeshv** [stern/stern#187] [General]
So Installed stern using kubectl krew install stern on my windows 10 laptop. It installed fine, and I can access it through kubectl-stern command. However, when I try to execute say kubectl-stern -A, I keep getting this error 
Error: One of pod-query, --selector, --field-selector or --prompt is required
Usage:
  stern pod-query [flags]
Do I have to specify pod-query with evey query?
I also tried specifying it kubectl-stern pod-query -A, but then this one just waits and does nothing for minutes. I have to forcibly kill it?
I'm executing a local KIND cluster for development. The cluster's up and running and I can execute kubectl commands against it and see the namespaces, and logs for the pods but doesn't work with kubectl-stern.
Is there a specific way of making it to work on Windows? Appreciate any help on this.
Thanks,
Asheesh

**Comment by superbrothers** [comment]
You must specify pod-query. This is the same for stern and kubectl-stern.

```
$ kubectl stern -A .
```

```
$ stern -A .
```

**Reply by asheeshv** [reply]
Thanks for the response superbrothers, so when you say pod-query, you don't mean that I need to add the words "pod-query" as part of kubectl stern commands? I tried executing the kubectl stern -A . and it fetched the logs for all the pods, so that worked, but I didn't have to add the words "pod-query".  Actually the usage is confusing to me:
the output for incorrect usage says to do this stern pod-query [flags]
So I though [flags] is the one that we need to set but we do need include pod-query with every other command.
Sorry for such noob questions. Trying to figure this out.
Thanks for your help!
